odl/solvers/findroot/newton.py

```diff
@@ -67,8 +67,8 @@ def bfgs_method(grad, x, line_search, niter=1, partial=None):
         Strategy to choose the step length
     niter : `int`, optional
         Number of iterations
-    partial : `Partial`, optional
-        Object executing code per iteration, e.g. plotting each iterate
+    callback : `callable`, optional
+        Object executing code per iteration, e.g. plotting each iterate.
```

Comment:
Made a separate issue on this #432 will leave this and make a general go at this once that is settled.